clickhouse-native-jdbc/src/test/java/com/github/housepower/jdbc/LowCardinalityTypeTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.github.housepower.jdbc;
+
+import com.github.housepower.misc.BytesHelper;
+import org.junit.jupiter.api.Test;
+
+import java.sql.DatabaseMetaData;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+//Refer to [[https://github.com/housepower/ClickHouse-Native-JDBC/issues/442]] for more details.
+public class LowCardinalityTypeTest extends AbstractITest implements BytesHelper {
+    @Test
+    public void testLowCardinalityType() throws Exception {
+        withStatement(statement -> {
+            statement.execute("DROP TABLE IF EXISTS low_cardinality_test");
+            statement.execute("CREATE TABLE IF NOT EXISTS low_cardinality_test " +
+                    "(value LowCardinality(String), nullable_value LowCardinality(Nullable(String))) Engine=Memory()");
+
+            String testValue = "value";
```

Comment:
Better to use multi values to test low_cardinality cause it'll create dictionary on the server side.

Reply:
done